src/main/scala/com.snowplowanalytics.sauna/actors/CommonActor.scala

```diff
@@ -55,6 +57,7 @@ abstract class CommonActor(respondersConfig: RespondersConfig,
   if (respondersConfig.recipientsEnabled) {
     responderActors +:= context.actorOf(Recipients(sendgrid), "Recipients")
   }
+    responderActors +:= context.actorOf(UAResponder(urbanairship), "UAResponder")
```

Comment:
Do we need to *always* launch Urban Airship responder? No configurations like for other responders?